modules/cache-common/src/internalClusterTest/java/org/opensearch/cache/common/tier/TieredSpilloverCacheStatsIT.java

```diff
@@ -498,4 +504,16 @@ private static ImmutableCacheStatsHolder getNodeCacheStatsResult(Client client,
         NodeCacheStats ncs = nodeStatsResponse.getNodes().get(0).getNodeCacheStats();
         return ncs.getStatsByCache(CacheType.INDICES_REQUEST_CACHE);
     }
+
+    // Check each stat separately for more transparency if there's a failure
```

Comment:
The problem with this approach is that if a new field is added to ImmutableCacheStats then it is possible this test will miss it and that field won't factor into the equality check here. Would simply adding a good toString() implementation on ImmutableCacheStats solve this problem?

Reply:
Yes, good idea. Will make that change